feat(upnp): add implementation based on IGD protocol (#4156)
paritytech/rust-libp2p: libp2p/CHANGELOG.md

- Add `json` feature which exposes `request_response::json`.
  See [PR 4188].

> - Add initial version of `upnp` protocol see [PR 4156]

Reviewer: ```suggestion
- Add support for UPnP via the IGD protocol.
  See [PR 4156].
```

Author: thanks, updated!
